main.rb

```diff
@@ -0,0 +1,218 @@
+require './student'
+require './teacher'
+require './book'
+require './person'
+require './rental'
+
+class Main # rubocop:disable Metrics/ClassLength
+  def initialize
+    @book_ist = []
+    @list_people = []
+    @rentals = []
+  end
+
+  def first_display
+    puts 'Welcome to School Library App!'
+    prompt_user
+  end
+
+  def prompt_user
+    options
+    input = gets.chomp
+
+    if input.to_i.between?(1, 6)
+      print "\e[2J\e[f"
+      selection(input)
+    elsif input == '0'
+      print "\e[2J\e[f"
+      nil
+    else
+      invalid_prompt
+      prompt_user
+    end
+  end
+
+  def options
+    puts 'Please choose an option by entering a number'
+    puts '1 - Create a user'
+    puts '2 - Create a book'
+    puts '3 - Create a rental'
+    puts '4 - List all books'
+    puts '5 - List all people'
+    puts '6 - List all rentals by user id'
```

Comment:
- [ ] Hi @Have-Samuel, kindly make sure that your app is working properly. 
- I can't create a user(Student of Teacher)
- I can't create a a book
- I can't equally create a rental
- **Screenshot:**

![Screenshot from 2022-06-17 18-33-53](https://user-images.githubusercontent.com/63087491/174350151-0980caf2-0b3f-445b-9fad-f17bfc0430ae.png)